### DISCUSSION TITLE
saf-training/courses/advanced/04

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#235] [Ideas]
# saf-training/courses/advanced/04

Resources Now that you have learned about making and running InSpec profiles, let's dig deeper into the mechanics of InSpec by learning about resources. Core Resources As you sa...

https://mitre.github.io/saf-training/courses/advanced/04.html

**Comment by aaronlippold** [comment]
Update the example code to have the correct imports at the top.
